rclpy/threading/data_plotter.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```

Comment:
It's true that won't run as you'd expect on Windows, the only portable way to do that is with a setuptools entry point, so that it creates an `.exe` file for the script on Windows. I think it's fine to leave it though, and Windows users will need to do `python3 path/to/data_plotter.py` instead of `./path/to/data_plotter.py`.

It should be fine, so long as we're not running this in an automated way on Windows.